crates/kubelet/src/image_client.rs

```diff
@@ -10,33 +10,63 @@ use oci_distribution::Reference;
 /// An image client capable of fetching images from a storage location
 #[async_trait]
 pub trait ImageClient {
```

Comment:
Not related to this PR necessarily - but the naming here is still a bit misleading, and can lead people to think these are actually images.